### DISCUSSION TITLE
regarding sale of the github swag coupons

### DISCUSSION THREAD

**Discussion by mridungupta1984** [community/community#43229] [GitHub Education] [answered]
I know a teacher who is selling out the github swag coupons at some website, and i want to report it as it is not allowed as per the terms of github teacher pack

**Comment by mridungupta1984** [comment]
@github-staff look into this ASAP

**Reply by mridungupta1984** [reply]
Actually I don't know about the name and school of the teacher, and I tried to mail Github support initially but they replied me to create a ticket, but they did not responded yet

**Reply by divais** [reply]
Can you please share the ticket number so that we can take it forward.

**Reply by mridungupta1984** [reply]
@divais Ticket number #1936504

**Reply by mridungupta1984** [reply]
> Can you please share the ticket number so that we can take it forward.

Thank you @divais for acknowledging the issue and sharing further to the team.

**Comment by mridungupta1984** [comment]
@queenofcorgis @courtneycl

**Comment by mridungupta1984** [comment, marked as answer]
https://github.com/orgs/community/discussions/43229?sort=top#discussioncomment-4743202

ticket created with github support. thank you

**Reply by mridungupta1984** [reply]
> Hello there mridung, can you provide details where the coupons are being sold. We can retrieve his information from the website.

store.hashhackers.com
